majors_and_certificates/certificates/AFS.yaml

```diff
@@ -0,0 +1,44 @@
+---
+type: Certificate
+name: African Studies
+code: AFS
+degree:
+urls:
+  - https://afs.princeton.edu/undergraduate-opportunities/certificate-african-studies
```

Comment:
This line doesn't need to be indented. The dash that designates it as a list already counts as an indentation. Not sure what the official style guide says, but this keeps it consistent with the rest of the lists in this file.